cron/update_geolite_db.py

```diff
@@ -60,9 +69,9 @@ def initConfig():
     options.sysloghostname = getConfig('sysloghostname', 'localhost', options.configfile)
     options.syslogport = getConfig('syslogport', 514, options.configfile)
 
-    options.db_download_location = getConfig('db_download_location', '', options.configfile)
-    options.db_location = getConfig('db_location', '', options.configfile)
-
+    options.db_store_location = getConfig('db_store_location', '', options.configfile)
```

Comment:
Here the options vars are renamed, but I don't see an associated config file in this PR to ensure these options are indeed renamed. 
Also to build this out in MozDef-deploy, is there an associated PR?

Reply:
Updated the example config file